toolkits/global/packages/global-popup/js/popup.js

```diff
@@ -95,22 +96,48 @@ const Popup = class {
 
 	_calcPositioning() {
 		const distanceScrolled = document.documentElement.scrollTop;
-		const triggerMetrics = this._trigger.getBoundingClientRect();
+		const triggerMetricsCollection = this._trigger.getClientRects();
```

Comment:
This is a jQuery method isn't it?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/getClientRects